src/sonic-config-engine/minigraph.py

```diff
@@ -2474,7 +2474,10 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
 
     # asymmetric PFC is disabled by default
     for port in ports.values():
-        port['pfc_asym'] = 'off'
+        if 'pfc_asym' not in port:
+            port['pfc_asym'] = 'off'
+        elif port['pfc_asym'] == '-':
+            del port['pfc_asym']
```

Comment:
For ports with pfc_asym "-", can we do port['pfc_asym'] = 'off'?

Reply:
No, those ports can't handle pfc_asym configuration.